cmd2.py

```diff
@@ -43,10 +44,38 @@
 import traceback
 import unittest
 from code import InteractiveConsole
+from enum import Enum
```

Comment:
The ``enum`` module was added in Python 3.4.  For Python 2.7, we either need to not use ``enum`` or add a conditional dependency on the ``enum34`` backport like so:

```Python
try:
    from enum34 import Enum
except ImportError:
    from enum import Enum
```